### PR TITLE
writing fix

### PR DESCRIPTION
**Pull request by szha**
## Description ##
writing fix

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] Code is well-documented

## Notes ##
Many updates in the notebooks are courtesy of @zackchase.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-301/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-301/3/index.html

File: docs/examples/language_model/language_model.ipynb [hunk 1]
```diff
@@ -24,13 +33,17 @@
    "source": [
     "## Language model definition - one sentence\n",
     "\n",
-    "A Language model tries to predict the next word given the previous ones.\n",
+    "The standard approach to language modeling consists of training a model that given a trailing window of text, predicts  the next word in the sequence. When we train the model we feed in the inputs $x1, x_2, ...$ and try at each time step to predict the corresponding next word $x_2, ..., x_{n+1}$. To generate text from a language model, we can iteratively predict the next word, and then feed this word as the input to the model at the subsequent time step.\n",
+    "\n",
+    "<div style=\"width: 500px;\">![Language Model](https://gluon.mxnet.io/_images/recurrent-lm.png)</div>\n",
```
Comment (leezu) [inline docs/examples/language_model/language_model.ipynb]: It may be better to directly use proper HTML:
``

Comment (szha) [reply docs/examples/language_model/language_model.ipynb]: I adopted this from Zack's notebook. Agreed, I can update.

Comment (leezu) [reply docs/examples/language_model/language_model.ipynb]: Ok, thanks.

**Comment by mli** [comment]
Job PR-301/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-301/10/index.html

**Pull request merged**